app/javascript/dashboard/routes/dashboard/conversation/contact/EditContact.vue

```diff
@@ -11,6 +11,8 @@
         :contact="contact"
         :in-progress="uiFlags.isUpdating"
         :on-submit="onSubmit"
+        @success="onCancel"
```

Comment:
`onSuccess`

Reply:
I prefer not to use `on` before emitted events. as `@success` is already short for `v-on:success`. @pranavrajs What do you think ?

Reply:
I meant that the callback is wrong. `@success` is fine.

Reply:
The idea was to call modal closing function after success, I'll create another function

Reply:
Made the changes